setup.py

```diff
@@ -4,11 +4,11 @@
 
 setup(
     name='dtreeviz',
-    version='1.1.4',
+    version='1.2',
     url='https://github.com/parrt/dtreeviz',
     license='MIT',
     packages=find_packages(),
-    install_requires=['graphviz>=0.9','pandas','numpy','scikit-learn',
+    install_requires=['graphviz>=0.9','Pillow','pandas','numpy','scikit-learn',
```

Comment:
I will put this comment here because I cannot on rf-partitioning.ipynb. Nice visualisations btw ;)

pltvid isn't installed and after installing it, I have this error :
<img width="601" alt="Screen Shot 2021-02-16 at 10 44 58 AM" src="https://user-images.githubusercontent.com/12815158/108039019-36898000-7044-11eb-9f92-1d3d02017485.png">

Reply:
I managed to solve this by "brew install poppler" (it took a while to install it).
Do you remember how you managed to make it work for yourself? because by installing the poppler using brew can be uncomfortable. I saw on stackoverflow by installing it using pip, but didn't work...

Reply:
actually I think that it is PIL that we need to install not Pillow, which is something else.